RELEASE_NOTES.md

```diff
@@ -3,27 +3,42 @@
 ## 4.5.0
 ### Features
 - The `podman kube play` command now supports the hostIPC field ([#17157](https://github.com/containers/podman/issues/17157)).
+- The `podman kube play` command now supports a new flag, `--wait`, that keeps the workload running in foreground until killed with a sigkill or sigterm. The workloads are cleaned up and removed when killed ([#14522](https://github.com/containers/podman/issues/14522)).
```

Comment:
```suggestion
- The `podman kube play` command now supports a new flag, `--wait`, that keeps the workload running in the foreground until killed with a sigkill or sigterm. The workloads are cleaned up and removed when killed ([#14522](https://github.com/containers/podman/issues/14522)).
```

Reply:
Sorry for getting to this so late. I realize this can't be fixed at this point, but, "The workloads are cleaned up" is obviously not true if we're talking about SIGKILL.